src/WorkbenchAccessManager.php

```diff
@@ -116,22 +142,26 @@ public static function getDefaultValue() {
    * {@inheritdoc}
    */
   public function getUserSections($uid = NULL, $add_roles = TRUE) {
```

Comment:
I think this method belongs on another service too, it has nothing to do with plugins, and hence breaks the single responsibility principle - so I opened https://www.drupal.org/node/2862725